pkg/volume/gce_pd/gce_pd_test.go

```diff
@@ -91,6 +96,16 @@ func (fake *fakePDManager) DeleteVolume(cd *gcePersistentDiskDeleter) error {
 	return nil
 }
 
+func getNodeSelectorRequirementWithKey(key string, term v1.NodeSelectorTerm) (*v1.NodeSelectorRequirement, error) {
+	for _, r := range term.MatchExpressions {
+		if r.Key != key {
```

Comment:
nit: use positive expressions when possible, especially here this feels a little backwards

instead, maybe:
```
if r.Key == key{
  return &r, nil
}
```